### PR TITLE
fix test error

### PR DESCRIPTION
**Pull request by intellinjun**
## Type of Change

feature or bug fix or documentation or others
API changed or not

## Description
[jira1188](https://jira.devtools.intel.com/browse/NLPTOOLKIU-1188)
detail description 
Issues: xxx
extension test

### CODE REVIEW

**Comment by intellinjun** [comment]
https://inteltf-jenk.sh.intel.com/job/neural_speed_extension/

File: tests/model-test/cpp_graph_inference.sh [hunk 1]
```diff
@@ -411,7 +411,7 @@ function main() {
                     else
                         real_ctx=$ctx # TODO(Zhenzhong): use same ctx for  chatglm & baichuan
                         [[ "${model}" == "chatglm2" || "${model}" == "chatglm-6b" ||
-                            "${model}" == "baichuan-13b" || "${model}" == "baichuan2-13b" ]] && real_ctx=2047
+                            "${model}" == "baichuan-13b" || "${model}" == "baichuan2-13b" ]]
 
                         NEURAL_SPEED_VERBOSE=1 OMP_NUM_THREADS=$cores_per_instance numactl -m 0 -C 0-$(($cores_per_instance - 1)) \
                             $infer_cmd --seed 1234 -t $cores_per_instance -b 2047 -c $real_ctx -n ${output} -m ${model}-${precision}.bin $extension -p "$prompt" 2>&1 | tee ${WORKSPACE}/${logs_file} || true &
```
Comment (DDEle) [inline tests/model-test/cpp_graph_inference.sh:417]: If the special `$real_ctx` is no longer necessary for chatglm & baichuan, we should just remove everything about `real_ctx` for clearity.
```suggestion
                        NEURAL_SPEED_VERBOSE=1 OMP_NUM_THREADS=$cores_per_instance numactl -m 0 -C 0-$(($cores_per_instance - 1)) \
                            $infer_cmd --seed 1234 -t $cores_per_instance -b 2047 -c $ctx -n ${output} -m ${model}-${precision}.bin $extension -p "$prompt" 2>&1 | tee ${WORKSPACE}/${logs_file} || true &
```

**Comment by intellinjun** [comment]
https://inteltf-jenk.sh.intel.com/job/neural_speed_extension/77/artifact/report.html

File: neural_speed/models/chatglm/chatglm.h [hunk 2]
```diff
@@ -26,7 +26,7 @@ enum chatglm_model {
 static const model_scratch chatglm_mem_req(int n_layers) {
   switch (n_layers) {
     case 28:
-      return {2048ull * MB, 2048ull * MB, 4096ull * MB};
+      return {4096ull * MB, 4096ull * MB, 8192ull * MB};
```
Comment (zhentaoyu) [inline neural_speed/models/chatglm/chatglm.h:29]: Does it still need the larger memory when `MAX_REQUEST_NUM=1`?

**Review by zhentaoyu — APPROVED** [review]

**Review by Zhenzhong1 — APPROVED** [review]

File: neural_speed/models/chatglm/chatglm.h (hunk 2 above)
Comment (Zhenzhong1) [reply neural_speed/models/chatglm/chatglm.h:29]: Chatglm2 claims that it supports 32k tokens as input. But a little less memory here is no problem.

**Pull request merged**